tests/Support/SupportStrTest.php

```diff
@@ -536,6 +536,15 @@ public function testUcfirst()
         $this->assertSame('Мама мыла раму', Str::ucfirst('мама мыла раму'));
     }
 
+    public function testOnlyNumbers()
+    {
+        $this->assertSame(123, Str::onlyNumbers("f1o2o3"));
+        $this->assertSame(45612, Str::onlyNumbers("U$ 456.12"));
```

Comment:
I would expect we would get `456.12` instead of `45612`

Reply:
also this would produce `143` for `Your balance is £-143`
and It would beautifully mangle exponential numbers.
and doesn't consider localised decimals  https://www.languageediting.com/format-numbers-eu-vs-us/


Perhaps the function should be called `onlyDigits`?  or `Str::digits` ?

Reply:
@powellblyth
You're right, I will change the function name to `Str::digits`, this will let the function more readable